.github/workflows/main_push_and_pull_request_workflow.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       matrix:
         java-version: [ 11 ]
-        test: [ 'LocalSystem', 'S3' ]
+        test: [ 'LocalSystem', 'S3', 'GCS' ]
```

Comment:
Locally failed with `GCS` "tests not found", maybe safer to use `Gcs`?

Reply:
The problem is worse...
Look at the "S3" E2E run
![image](https://github.com/Aiven-Open/tiered-storage-for-apache-kafka/assets/2189030/16756e5c-c735-477e-a9ce-71c30a4a6e55)

I'll fix this separately.

Reply:
Here it is: https://github.com/Aiven-Open/tiered-storage-for-apache-kafka/pull/412

Reply:
🤦🏽

Reply:
Should be good now